doc/source/tools/postprocessing/postprocessing_paraview.rst

```diff
@@ -0,0 +1,51 @@
+========================================
+Massive Data Visualization with Paraview
+========================================
+
+Lethe is developed with special care for scalability; it is intended to be used for solving extremely large problems by using massively parallel computing. As such, the output files that are obtained can sometimes be so heavy that the post-processing requires special considerations. This section covers key points to take into account in such cases.

```

Comment:
I think this paragraph lacks a more specific sentence about the purpose of the guide, such as: When simulations are carried out on servers, it can be unpractical to download all results...

Reply:
I would also add a note here referring to paraview's guide